### PR TITLE
Documentation: Information on how to use HTTP API

### PR DESCRIPTION
**Pull request by tbroden84**
### WHY are these changes introduced?

This provider don't support the use of Message Broker HTTP API, but can be used together with other providers to do so.

### WHAT is this pull request doing?

Add information and example on how to use other provider to access Message Broker HTTP API.

### HOW can this pull request be tested?

Check new guide.md file.

### CODE REVIEW

**Comment by tbroden84** [comment]
Reason for failing checks: Go-api v1.16.0 been added without correct updates. Present in #257

**Comment by dentarg** [comment]
Can use https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs when changing only markdown :)

File: docs/guides/info_http_api.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+layout: "cloudamqp"
+page_title: "HTTP API"
```
Comment (dentarg) [inline docs/guides/info_http_api.md:3]: to make it more clear as just "HTTP API" is very generic term (needs updating in more places)

```suggestion
page_title: "Message Broker HTTP API"
```

**Review by dentarg — APPROVED** [review]
Nice

File: docs/guides/info_http_api.md (hunk 1 above)
Comment (tbroden84) [reply docs/guides/info_http_api.md:3]: Good point.

**Pull request merged**